velox/exec/fuzzer/PrestoQueryRunner.cpp

```diff
@@ -150,10 +150,10 @@ class ServerResponse {
 PrestoQueryRunner::PrestoQueryRunner(
     std::string coordinatorUri,
     std::string user,
-    std::chrono::milliseconds timeout)
+    uint32_t timeout)
```

Comment:
curiously why use uint32 here

Reply:
I wanted to use a non-negative integer here and presumed a timeout larger than `uint32_t::max()` would not be used. Please let me know if I should change it to `int64_t`.

Reply:
Can we keep `std::chrono::milliseconds` here and cast the timeout param to `std::chrono::milliseconds` before constructing the PrestoQueryRunner